typescript/nomad-deploy/src/incremental/index.ts

```diff
@@ -0,0 +1,69 @@
+import * as ethers from 'ethers';
+
+import { NomadContext } from '@nomad-xyz/sdk/src';
+import { toBytes32 } from '../utils';
+
+function toCall(t: ethers.ethers.PopulatedTransaction): {to: string, data: string} {
+  return { to: toBytes32(t.to!), data: t.data! };
```

Comment:
I would prefer if the conversion were done in the `sdk` but that can happen in a separate PR